Makefile

```diff
@@ -63,7 +63,7 @@ update-man.rst:
 	$(MAKE) generate-man ARGS="-o docs/source/man.rst"
 
 manpage-test:
-	@poetry run $(MAKE) --no-print-directory generate-man | diff -u docs/source/man.rst -
```

Comment:
Don't forget to put this back, assuming this was a temporary removal given the commit was called "test".